composer.json

```diff
@@ -1,9 +1,9 @@
 {
     "name": "cron/neos-behat",
-    "type": "typo3-flow-package",
+    "type": "neos-package",
     "description": "Neos Behat Addons",
     "require": {
-        "typo3/flow": "*"
+        "neos/flow": "*"
```

Comment:
@Magarif After this PR, we require at least flow 4.0.0, so we should mark that explicitly:

```
    "neos/flow": "^4.0.0"
```

I will add this to the PR now.